blog/2025/pcsx2-2.4_2.2/index.mdx

```diff
@@ -0,0 +1,216 @@
+
+## PCSX2 2.2.0
+
+PCSX2 2.0 may well have been [our biggest single release to date](../../2024/pcsx2-2-release/index.mdx). With several major contributors stepping back, including longtime PCSX2 lead developer refraction, the remaining team – still energized to contribute – confronted reality: for the first time in a long time, we'd delivered what we had to offer in the way of big, bombastic new features. What we resolved for PCSX2 2.2 was to take it out of the furnace, let it cool, and hammer it into something more refined before getting up to speed again in 2.4. For this period of a few months, the word on everyone's mind was "polish".
+
+<PCSX2PRLink
+  authors="Florin9doi,joestringer"
+  prNums="11614,11616,11617,11630,11633,11715,11723,12157"
+>
+
+### New Emulated Peripherals
+
+</PCSX2PRLink>
+
+Almost immediately after 2.0, contributor Florin9doi got to work chipping away at the list of peripherals we haven't implemented yet. Not long after, first-time contributor joestringer began working on train controller support. Thanks to them, in just about the first month of 2.0, we now had support for
+
+These are:
+
+- [Trance Vibrator](https://www.theverge.com/2016/9/25/13046770/rez-trance-vibrator-ps2) –
+- [Gametrak/_RealPlay_](https://en.wikipedia.org/wiki/Gametrak) device –
+- [Jogcon](https://en.wikipedia.org/wiki/Jogcon) and [NeGcon](https://en.wikipedia.org/wiki/NeGcon) –
+- [Train Mascon](https://marcriera.github.io/ddgo-controller-docs/controllers/usb/cotm02001/) controllers (Type 2, Shinkansen, and Ryojōhen) –
+- Konami Microphone –
+- [Zip 100](https://en.wikipedia.org/wiki/Zip_drive) –
+- [PictureParadise](https://www.ign.com/articles/2000/09/26/tgs-2000-put-yourself-in-the-game) –
+
+On top of that, PCSX2's [EyeToy](https://en.wikipedia.org/wiki/EyeToy) emulation now has audio support, and [_Buzz!_](https://en.wikipedia.org/wiki/Buzz!) devices now have a custom UI!
+
+<Image src={require("./img/controller_ryojohen.webp").default} />
+<Image src={require("./img/controller_jogcon.webp").default} />
+
+<PCSX2PRLink authors="lightningterror" prNums="11737,11748">
+  ### Will It Blend?
+</PCSX2PRLink>
+
+PCSX2 team member and graphics expert LightningTerror has implemented per-pixel [alpha blending](https://en.wikipedia.org/wiki/Alpha_compositing) (PABE), improving both accuracy and speed. Software blending, run on the CPU, is highly accurate but incurs a performance hit as it can't take advantage of the parallelization inherent in graphics work. Hardware blending, run on the GPU, is highly efficient for this reason but can suffer in accuracy if extreme care isn't taken. Eliminating the need to run blending on the CPU, PABE blends (heh) the accuracy from software blending with the efficiency of hardware blending. The benchmarks below provided by KamFretoZ and JordanTheToast show the considerable speedup PABE brings to both AMD and Nvidia. But that wasn't the only improvement!
```

Comment:
He hasn't "implemented" it, he just added it to a couple of specific scenarios it didn't turn on for previously.